x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -93,19 +96,30 @@ export const createLogThresholdExecutor = (libs: InfraBackendLibs) =>
     LogThresholdAlertState,
     LogThresholdAlertContext,
     LogThresholdActionGroups
-  >(async ({ services, params }) => {
-    const { alertWithLifecycle, savedObjectsClient, scopedClusterClient } = services;
-    const { sources } = libs;
-    const alertFactory: LogThresholdAlertFactory = (id, reason, value, threshold) =>
-      alertWithLifecycle({
+  >(async ({ services, params, startedAt }) => {
+    const { alertWithLifecycle, savedObjectsClient, scopedClusterClient, getAlertStartedDate } =
+      services;
+    const { sources, basePath } = libs;
+    let indexedStartedAt: string = startedAt.toISOString();
+    const alertFactory: LogThresholdAlertFactory = (id, reason, value, threshold) => {
+      indexedStartedAt = getAlertStartedDate(id) ?? indexedStartedAt;
+      return alertWithLifecycle({
         id,
         fields: {
           [ALERT_EVALUATION_THRESHOLD]: threshold,
           [ALERT_EVALUATION_VALUE]: value,
           [ALERT_REASON]: reason,
         },
       });
+    };
 
+    const relativeViewInAppUrl = getLogsAppAlertUrl(new Date(indexedStartedAt).getTime());
+    const viewInAppUrl = basePath.publicBaseUrl
+      ? new URL(
+          join(basePath.serverBasePath, relativeViewInAppUrl),
```

Comment:
Doesn't node's `path.join` use the platform-specific separator? Does this still work on Windows?

Reply:
Done